project-skeletons/elisp-package-with-docs/Makefile

```diff
@@ -58,3 +58,12 @@ $(TEXI_MANUAL) : $(PKG_DIR) $(ORG_MANUAL)
 	$(CASK) exec $(EMACSBATCH) \
 	-l org -l ox-texinfo \
 	--file=$(ORG_MANUAL) -f org-texinfo-export-to-texinfo
+
+
+test: unit ecukes
+
+unit:
```

Comment:
You'll need to add `test`, `unit` and `ecukes` to the `.PHONY` targets at the top. Also, you can delete the `check` task since these are better. :)